README.md

```diff
@@ -1,56 +1,128 @@
 [![Maven Central](https://img.shields.io/maven-central/v/io.openliberty/openliberty-runtime.svg?label=Maven%20Central)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22io.openliberty%22%20a%3A%22openliberty-runtime%22)
 
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://openliberty.io/">
+    <img src="https://openliberty.io/img/spaceship.svg" alt="Logo">
+  </a>
+</p>
+<p align="center">
+  <a href="https://openliberty.io/">
+    <img src="https://github.com/OpenLiberty/open-liberty/blob/master/logos/logo_horizontal_light_navy.png" alt="title" width="400">
+  </a>
+</p>
+
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+* [Summary](#summary)
+* [Getting Started](#getting-started)
+  * [Prerequisites](#prerequisites)
+  * [Downloads](#downloads)
+* [Usage](#usage)
+  * [Docs](#open-liberty-docs)
+* [Contributing](#contributing)
+  * [Running a Build](#running-a-build)
+* [License](#license)
+* [Contact](#contact)
+
 # Summary
-Open Liberty is a highly composable, fast to start, dynamic application server runtime environment.
+A lightweight open framework for building fast and efficient cloud-native Java microservices.
 
-# Table of Contents
-* [Getting Started](https://github.com/OpenLiberty/open-liberty#getting-started)
-* [Contribute to Open Liberty](https://github.com/OpenLiberty/open-liberty#contribute-to-open-liberty)
-* [Community](https://github.com/OpenLiberty/open-liberty#community)
+* The Open Liberty modular architecture means only the features you need are loaded at runtime to help reduce image size and memory consumption. The smaller footprint means you can run more application instances per machine to help reduce production costs.
 
-## Getting Started
+* Servers running Open Liberty start faster, making them ideal for rapid scale-up and scale-down of containerized applications.
 
-### Downloading
-You can download released versions and nightly build artifacts from: https://www.openliberty.io/downloads/
+## Getting Started
 
-### Quick start
+### Launch a sample app using [Maven](https://maven.apache.org/)
     git clone https://github.com/OpenLiberty/sample-getting-started.git
     cd sample-getting-started
     mvn clean package liberty:run-server
 
 Open browser to http://localhost:9080/ and explore the demo application.
 
-### Open Liberty Guides
-Visit the [OpenLiberty website](https://openliberty.io/guides/) for a number of step by step guides.
+This [guide](https://openliberty.io/guides/getting-started.html) shows how it was built
+
+### Prerequisites
+
+* Java SE 8,11,12 - [More details](https://openliberty.io/docs/ref/general/#java-se.html)
+
+### Downloads
+
+* Zipped released versions and nightly build artifacts from: https://www.openliberty.io/downloads/
+* Official Released Open Liberty Docker containers from : https://hub.docker.com/_/open-liberty
+
+## Usage
+
+### Controlling Open Liberty
```

Comment:
Most users don't directly control Liberty by invoking the server script. Typically tooling like the Maven/Gradle plugins, WDT, or Docker containers handle this stuff. Perhaps we could link to our Maven/Gradle/Docker guides here instead?

Reply:
This will take more thought - At the very least we need links to Docker Usage, Maven Plugin Usage, and command line Usage docs....   Its not obvious...  But I still think we need to help the user who builds locally, and wants to run some tests on the command line.